build.sbt

```diff
@@ -10,9 +10,11 @@ crossScalaVersions := Seq("2.12.12", "2.13.3")
 
 libraryDependencies ++= Seq(
     "com.typesafe.akka" %% "akka-actor" % "2.6.0" % "provided",
```

Comment:
Can you set the versions on the current akka dependencies to `2.6.10` as well?

Reply:
Sure, I didn't set it to `2.6.10` because I was afraid to broke something. 😆